src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,21 @@
+.card__stars {
+  display: flex;
+  justify-content: space-between;
+
+  &-stars {
+    background-image: url(/src/images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+    width: 16px;
+    height: 16px;
+    margin-left: 4px;
+  }
+
+  &-stars:nth-child(-n + 4) {
+    background-image: url(/src/images/star-active.svg);
+  }
```

Comment:
The same issue with the class name here. It should be `&__stars:nth-child(-n + 4)`.